src/service/types.ts

```diff
@@ -68,6 +68,7 @@ export type DepartureFavoritesPayload = {
 export type DepartureFavoritesQuery = CursoredQuery<{
   startTime: Date;
   limitPerLine: number;
+  includeCancelledTrips: boolean;
```

Comment:
Needed when using this type instead of `DepartureGroupsQuery` in departure favorites.